Makefile

```diff
@@ -108,3 +108,12 @@ apidoc:
 apidoc_clean:
 	rm -rf apidoc
 endif
+
+## Editor navigation files
+C_SOURCE_FILES = $(wildcard include/*/*.h library/*.[hc] programs/*/*.[hc] tests/suites/*.function)
+tags: $(C_SOURCE_FILES)
+	ctags -o TAGS $(C_SOURCE_FILES)
+TAGS: $(C_SOURCE_FILES)
+	etags -o TAGS $(C_SOURCE_FILES)
+GPATH GRTAGS GSYMS GTAGS: $(C_SOURCE_FILES)
+	 ls $(C_SOURCE_FILES) | gtags -f - --gtagsconf .globalrc
```

Comment:
There is an extra space after the tab before `ls`.

Reply:
Do you mind if I fix the offending commit by rebasing? I'll need to push another bug fix anyway, `compat.sh` is still failing.

Reply:
I don't mind. Please feel free to rebase.